app/views/devise/registrations/edit.html.haml

```diff
@@ -15,7 +15,10 @@
           %h4
             Currently the following openIDs are associated with your account
           - @openids.each do |openid|
-            %li= "#{openid.provider}:#{openid.email}"
+            %li
+              %span "#{openid.provider}:#{openid.email}"
+              = link_to user_openid_path(openid, user_id: current_user.id), method: :delete do
+                %span.fa.fa-times
```

Comment:
The list appears a bit funny with the `X` moved to the left. 
We could remove the list bullets and use only the `X` (having both list-style role and a button role)

![screenshot from 2017-06-11 15-10-12_c](https://user-images.githubusercontent.com/17273872/27010931-db9be418-4eb8-11e7-967e-2aa485df29db.png)
![screenshot from 2017-06-11 15-13-16](https://user-images.githubusercontent.com/17273872/27010932-db9f3cee-4eb8-11e7-8be7-9d6f08f7e38c.png)